index.js

```diff
@@ -95,7 +95,8 @@ app.post('/github-hook', function (req, res, next) {
             var u = (issue.user && issue.user.login) || null;
             var content = issue.body || "";
             if (!isComment && action == "edited") {
-                if (body.pull_request.merged) return; // we know .pullrequest is available because it's not a comment.
+                // we know .pull_request is available because it's not a comment.
+                if (body.pull_request.draft || body.pull_request.merged) return;
```

Comment:
[This filtering is already implemented above.](https://github.com/web-platform-tests/wpt-pr-bot/blob/724e1bbad08db684f523b3bc7d72e226cdaeabcc/index.js#L85-L88)

Reply:
Yes, but something doesn't work :)

With a sample of the webhook payload that gets past all the filters it'd be easy to fix this, now I'm just trying all the places I can see.